File: src/backend/DAL/Entities/Alternative.cs
using System;
using System.Collections.Generic;

namespace DAL
{
    public class Alternative
    {
        public int AlternativeId { get; set; }

        public string AlternativeName { get; set; }

        public float GlobalPriority { get; set; }

        public DateTime DateCreated { get; set; }

        public DateTime DateUpdated { get; set; }

        public List<Criterium_Alternative> criterium_alternatives { get; set; }

    }
}

File: src/frontend/angular template/angular-crash-todolist/src/app/components/models/alternative.ts
type Priority={
    [key:number]:-9 | -7 | -5 | -3 | -1 | 1 | 3 | 5 | 7 | 9;
}
export class Alternative{
    AlternativeId:number;
    AlternativeName:string;
    completed:boolean;
    GlobalPriority:Priority;
}
